base_uom_merge/tests/test_base_uom_merge.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+
+
+class TestMergeUOM(TransactionCase):
+    def setUp(self):
```

Comment:
Change to setUpClass according to the migration rules to v15

Reply:
Updated, Thanks